### ISSUE TITLE
Maven plugin fails with 1.14.0

### ISSUE THREAD

**Issue by binkley** [gantsign/ktlint-maven-plugin#485]
I waited to post this issue to see if it would get resolved.  All my Kotlin projects trying 1.14.0 fail but pass with 1.13.1.

This failure mode is consistent for me across projects:
```
A type incompatibility occurred while executing com.github.gantsign.maven:ktlint-maven-plugin:1.14.0:format: class java.lang.String cannot be cast to class com.pinterest.ktlint.core.api.DefaultEditorConfigProperties$CodeStyleValue (java.lang.String is in module java.base of loader 'bootstrap'; com.pinterest.ktlint.core.api.DefaultEditorConfigProperties$CodeStyleValue is in unnamed module of loader org.codehaus.plexus.classworlds.realm.ClassRealm @75361cf6)
```

Java 17 is common among the projects, if relevant. (I don't have a sample project to check differential behavior between JDK 11 and 17 with plugin version 1.14.0).

**Comment by binkley** [comment]
0.15.0 is working for me, and I no longer see the above error.

**Comment by freemanjp** [comment]
@binkley thanks for reporting this issue and that it was resolved by version 1.15.0.
